pulsar-client-cpp/lib/checksum/crc32c_arm.h

```diff
@@ -37,11 +37,11 @@
 #define crc32c_u16(crc, v) __crc32ch(crc, v)
 #define crc32c_u32(crc, v) __crc32cw(crc, v)
 #define crc32c_u64(crc, v) __crc32cd(crc, v)
-#define PREF4X64L1(buffer, PREF_OFFSET, ITR)                                                                \
```

Comment:
`make format` wanted this diff; perhaps because I installed a newer/older version of clang-format? Regardless, it looks harmless.

Reply:
@zbentley Could you revert these changes first?

Reply:
@BewareMyPower sure, but as I recall, if I do then the clang-format build fails.

Reply:
Yeah. The `clang-format` version required is now 6 or later. You can simply run `docker-format.sh` to reformat the code.

Reply:
Done!